server/neptune/workflows/internal/rebase/workflow.go

```diff
@@ -0,0 +1,46 @@
+package rebase
+
+import (
+	"time"
+
+	"github.com/runatlantis/atlantis/server/neptune/workflows/activities"
+	"github.com/runatlantis/atlantis/server/neptune/workflows/activities/github"
+	"github.com/runatlantis/atlantis/server/neptune/workflows/activities/terraform"
+	workflowMetrics "github.com/runatlantis/atlantis/server/neptune/workflows/internal/metrics"
+	"go.temporal.io/sdk/temporal"
+	"go.temporal.io/sdk/workflow"
+)
+
+const (
+	RebaseGithubRetryCount    = 3
+	RebaseStartToCloseTimeout = 10 * time.Second
+)
+
+type Request struct {
+	Repo github.Repo
+	Root terraform.Root
+}
+
+type rebaseActivities struct {
+	*activities.RevsionSetter
+	*activities.Github
+}
```

Comment:
This should just be an interface, similar to the other workflows, you shouldn't have to depend on this directly.